fairlearn/reductions/_moments/moment.py

```diff
@@ -25,31 +24,50 @@ class Moment:
     def __init__(self):
         self.data_loaded = False
 
-    def load_data(self, X, y, **kwargs):
+    def load_data(self,
+                  X,
+                  y: pd.Series,
+                  *,
+                  sensitive_features: pd.Series = None):
         """Load a set of data for use by this object.
 
-        The keyword arguments can contain a :code:`sensitive_features` array.
-
-        :param X: The feature data
-        :type X: array
-
-        :param y: The true label data
-        :type y: array
+        Parameters
+        ----------
+        X : array
+            The feature array
+        y : pandas.Series
+            The label vector
+        sensitive_features : pandas.Series
+            The sensitive feature vector (default None)
         """
         assert self.data_loaded is False, \
             "data can be loaded only once"
-        self.X = X
+        assert isinstance(y, pd.Series)
+        if sensitive_features is not None:
+            assert isinstance(sensitive_features, pd.Series)
+        self._X = X
+        self._y = y
         self.tags = pd.DataFrame({_LABEL: y})
-        if _KW_SENSITIVE_FEATURES in kwargs:
-            self.tags[_GROUP_ID] = kwargs[_KW_SENSITIVE_FEATURES]
+        if sensitive_features is not None:
+            self.tags[_GROUP_ID] = sensitive_features
         self.data_loaded = True
         self._gamma_descr = None
 
+    @property
+    def X(self):
+        """Return the feature array as supplied to :meth:`load_data`."""
+        return self._X
+
     @property
     def total_samples(self):
         """Return the number of samples in the data."""
         return self.X.shape[0]
 
+    @property
+    def y_reformat(self):
```

Comment:
let's rename this to something that begins with `_` so that this is not part of the exported interface.

Reply:
`reformat` is a verb, so this indicates to me that it does something, rather than just return the reformatTED `y`.

Reply:
As noted above, this is now `_y_as_series` to be more descriptive.